src/main/java/redis/clients/jedis/BuilderFactory.java

```diff
@@ -2004,6 +2004,19 @@ public String toString() {
     }
   };
 
+  public static final Builder<TrackingInfo> TRACKING_INFO_RESPONSE = new Builder<TrackingInfo>() {
```

Comment:
For this class, a builder in either `TrackingInfo` or here (`BuilderFactory`) is enough. Two builders in both places is unnecessary.

Reply:
Hey @sazzad16 thanks for the review! I have updated the pr accordingly 👍